composer.json

```diff
@@ -42,7 +42,7 @@
         "twig/twig": "^2.12.1 || ^3.0"
     },
     "require-dev": {
-        "dama/doctrine-test-bundle": "^7.1",
+        "dama/doctrine-test-bundle": "^6.7",
```

Comment:
Curious why ?

Reply:
Class 'Doctrine\DBAL\Tools\Console\Helper\ConnectionHelper' not found

on lowest deps, something on the lower side of deps is not correctly defined. I might need to take a deeper look.

Reply:
But for now, all other sonata bundles uses 6.7 so we can keep with it a little while.